charts/wasmcloud-platform/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+1. You can access the wasmCloud dashboard with the following steps:
```

Comment:
What's the dashboard this is referring to?

If the goal is to use washboard (`wash ui`), then I think these instructions should be updated to demonstrated how that can be accessed, but otherwise I'm not sure these instructions are necessary.

If anything, it might be helpful to talk about how to connect (via `kubectl port-forward` or otherwise) with `wash` to interact with the cluster via NATS.

Reply:
I revised the NOTES.txt; please take a second look.